data/pathfinder/paizo/roleplaying_game/advanced_class_guide/acg_profs_weapon.lst

```diff
@@ -73,7 +73,7 @@ Taiaha.MOD				TYPE:Weapon Group Melee OneHanded Piercing
 Double Chicken Saber.MOD	TYPE:Weapon Group Melee OneHanded Slashing
 Broadsword (Nine Ring).MOD	TYPE:Weapon Group Melee OneHanded Slashing
 Terbutje.MOD			TYPE:Weapon Group Melee OneHanded Slashing
-Great Terbutje.MOD	TYPE:Weapon Group Melee OneHanded Slashing
+Terbutje (Great).MOD		TYPE:Weapon Group Melee OneHanded Slashing
```

Comment:
Is this weapon listed as "Terbutje, Great" somewhere? That would be the only reason to alter the KEY here.